src/Umbraco.Web.UI.Client/src/views/components/users/change-password.html

```diff
@@ -6,19 +6,15 @@
     </div>
     <div ng-switch="vm.changing">
         <div ng-switch-when="false">
-            <a href="" ng-click="vm.doChange()" class="btn btn-small">
+            <a href="" ng-click="vm.doChange()" class="btn umb-button__button btn-action umb-button--">
                 <localize key="general_changePassword">Change password</localize>
             </a>
         </div>
         <div ng-switch-when="true">
 
             <ng-form name="changePasswordForm">
                 <umb-control-group alias="resetPassword" label="@user_resetPassword" ng-show="vm.config.enableReset">
-                    <input type="checkbox" ng-model="vm.passwordValues.reset"
-                           name="resetPassword"
-                           val-server-field="resetPassword"
-                           no-dirty-check
-                           ng-change="vm.showReset = !vm.showReset" />
+                    <umb-checkbox  model="vm.passwordValues.reset" />
```

Comment:
@poornimanayar I think you've forgotten to include some of the previous attributes like:
`<umb-checkbox  model="vm.passwordValues.reset" name="resetPassword" val-server-field="resetPassword" on-change="vm.showReset = !vm.showReset"/>`

Reply:
@BatJan how about `no-dirty-check` attribute?

Reply:
@elit0451 I'll need to have a look at the directive later. It's been extended a bit since I initially introduced it to core so don't have the full overview of what's possible and not 😅 

Apart from that the name attribute should not be necessary though since angularjs only cares about the model value @poornimanayar 👍

Reply:
@elit0451 @poornimanayar Just had a look at the file and the no-dirty-check directive is probably not needed since the checkbox should accept a boolean value.